action.yml

```diff
@@ -16,6 +16,11 @@ inputs:
     description: "Github token with actions:read permission"
   jobStatus:
     description: "Give me job.status please!"
+  name:
+    description: "Defaults to job name"
```

Comment:
If you don't provide a default, isn't this then a required option?

Reply:
I want it to default to the job name per the job context (like it did before) but this value cannot be specified in the yaml definition file.